backend/app/views/spree/admin/shared/_product_tabs.html.erb

```diff
@@ -4,24 +4,11 @@
 <% content_for :tabs do %>
   <nav>
     <ul class="tabs" data-hook="admin_product_tabs">
-      <%= content_tag :li, class: ('active' if current == 'Product Details') do %>
-        <%= link_to t('spree.product_details'), spree.edit_admin_product_url(@product) %>
-      <% end if can?(:admin, Spree::Product) %>
-      <%= content_tag :li, class: ('active' if current == 'Images') do %>
-        <%= link_to t('spree.images'), spree.admin_product_images_url(@product) %>
-      <% end if can?(:admin, Spree::Image) && !@product.deleted? %>
-      <%= content_tag :li, class: ('active' if current == 'Variants') do %>
-        <%= link_to plural_resource_name(Spree::Variant), spree.admin_product_variants_url(@product) %>
-      <% end if can?(:admin, Spree::Variant) %>
-      <%= content_tag :li, class: ('active' if current == 'Prices') do %>
-        <%= link_to plural_resource_name(Spree::Price), spree.admin_product_prices_url(@product) %>
-      <% end if can?(:admin, Spree::Price) && !@product.deleted? %>
-      <%= content_tag :li, class: ('active' if current == 'Product Properties') do %>
-        <%= link_to plural_resource_name(Spree::ProductProperty), spree.admin_product_product_properties_url(@product) %>
-      <% end if can?(:admin, Spree::ProductProperty) && !@product.deleted? %>
-      <%= content_tag :li, class: ('active' if current == 'Stock Management') do %>
-        <%= link_to t('spree.stock_management'), spree.admin_product_stock_url(@product) %>
-      <% end if can?(:admin, Spree::StockItem) && !@product.deleted? %>
+      <% product_tabs(current: current).each do |tab| %>
```

Comment:
Could we pass the `product: @product` in here so we do not need to use and assign instance variables on the view context?